src/inlineSuggestions/textListener.ts

```diff
@@ -18,8 +21,18 @@ import getCurrentPosition, {
   isEmptyLinesWithNewlineAutoInsert,
   isOnlyWhitespaces,
 } from "./positionExtracter";
+import { AutocompleteResult } from "../binary/requests/requests";
 
-const EMPTY_LINE_WARMUP_MILLIS = 110;
+const EMPTY_LINE_WARMUP_MILLIS = 250;
+const EMPTY_LINE_DEBOUNCE = 550;
+
+const debouncedEmptyLinesRequest = debounce(
```

Comment:
debounce is not promise aware, https://github.com/component/debounce/blob/8be734c4b5b5a58a517f2441e0d90242a4a564cd/index.js#L27
meaning you cant pass an async function(at least not without handling rejections)